cmd/contour/ingressstatus.go

```diff
@@ -108,24 +109,30 @@ func (isw *loadBalancerStatusWriter) Start(stop <-chan struct{}) error {
 
 func parseStatusFlag(status string) v1.LoadBalancerStatus {
 
-	// Use the parseability by net.ParseIP as a signal, since we need
-	// to pass a string into the v1.LoadBalancerIngress anyway.
-	if ip := net.ParseIP(status); ip != nil {
-		return v1.LoadBalancerStatus{
-			Ingress: []v1.LoadBalancerIngress{
-				{
-					IP: status,
-				},
-			},
+	// Support ','-separated lists
```

Comment:
Nit. Could you please end comments with a full stop?

Reply:
Done.